main/commons/src/main/java/org/cryptomator/common/settings/VaultSettings.java

```diff
@@ -54,7 +54,8 @@
 	private final StringProperty mountFlags = new SimpleStringProperty(DEFAULT_MOUNT_FLAGS);
 	private final IntegerProperty filenameLengthLimit = new SimpleIntegerProperty(DEFAULT_FILENAME_LENGTH_LIMIT);
 	private final ObjectProperty<WhenUnlocked> actionAfterUnlock = new SimpleObjectProperty<>(DEFAULT_ACTION_AFTER_UNLOCK);
-
+	private final BooleanProperty lockAfterTime = new SimpleBooleanProperty(DEFAULT_LOCK_AFTER_TIME);
+	private final StringProperty lockTimeInMinutes = new SimpleStringProperty(DEFAULT_LOCK_TIME_IN_MINUTES);
```

Comment:
Shouldn't this be some numeric value?

Reply:
Yes, at first I didn't know one can add a Converter when bindingBidirectional.
Should we just stick with integers?
Then we don't need to differentiate between decimal point and decimal comma in localization.

Reply:
Yup, but the base unit should be seconds here (can be minutes in UI, of course - now that you know of custom converters^^).